### ISSUE TITLE
Update and release bundle that uses com.newrelic.telemetry:opentelemetry-exporters-newrelic-auto:0.12.1

### ISSUE THREAD

**Issue by jasonjkeller** [newrelic/newrelic-opentelemetry-integration-java#45]
[io.opentelemetry.javaagent:opentelemetry-javaagent-spi:0.12.1](https://repo1.maven.org/maven2/io/opentelemetry/javaagent/opentelemetry-javaagent-spi/0.12.1/) was recently released and will require an update to the OTel exporter. Once that work is complete we should update this bundle to use the new artifact `com.newrelic.telemetry:opentelemetry-exporters-newrelic-auto:0.12.1` and publish a new bundle artifact.

This issue is blocked by: https://github.com/newrelic/opentelemetry-exporter-java/issues/142

**Comment by jasonjkeller** [comment]
Closing as this story was a dupe of work that is already done.
